update tilemap doc (#1647)
cocos-creator/creator-docs: en/asset/tiledmap.md

## Storage in the Project

> For efficient asset management, it is recommended to store the imported `.tmx`, `.tsx` and `.png` files in a separate directory, not mixed with other assets. Highly recommended to put the `tmx` file and the `tsx` file in the same directory, otherwise the resources may not be loaded correctly.

Reviewer: ```suggestion
For efficient asset management, it is recommended to store the imported `.tmx`, `.tsx` and `.png` files in a separate directory and not mixed with other assets. It is also highly recommended to put the `.tmx` file and the `.tsx` file in the same directory, otherwise the resources may not be loaded correctly.
```

> For efficient asset management, it is recommended to store the imported `.tmx`, `.tsx` and `.png` files in a separate directory and not mixed with other assets. It is also highly recommended to put the `.tmx` file and the `.tsx` file in the same directory, otherwise the resources may not be loaded correctly.

Reviewer 2: ```suggestion
For efficient asset management, it is recommended to store the imported `.tmx`, `.tsx` and `.png` files in a separate directory and not mixed with other assets. Note that it is important to manage `tmx` files and `tsx` files in the same directory, otherwise the assets may not be loaded correctly.
```
